grpc/testing/test.proto

```diff
@@ -95,6 +95,7 @@ service LoadBalancerStatsService {
 // with HookRequestCommand::START
 service HookService {
   rpc Hook(grpc.testing.Empty) returns (grpc.testing.Empty);
+  rpc ReturnAll(ReturnAllRequest) returns (grpc.testing.Empty);
```

Comment:
Why is this needed? This is different from what was in the design.

Reply:
I feel like it is more logical. Current plan is to move the hook service to the maintenance port and have it run all the time. This makes START and STOP hook requests obsolete, leaving only the Return one. I really don't see a need to couple XdsHealthService to the HookService just to forward the Return command.

Reply:
Okay, discussed offline. The Hook command function and the actual Hook function are combined into a single service. Sounds good.

Reply:
But what about the `reset` functionality described in the doc? Since `ReturnAll` also applies to future `Hook` calls if there is a need to reset this state we should have it, no?

Reply:
Added ClearReturnStatus